mayastor/src/lvs/lvs_pool.rs

```diff
@@ -243,25 +257,130 @@ impl Lvs {
         }
     }
 
+    /// imports a pool based on its name, uuid and base bdev name
+    #[tracing::instrument(level = "debug", err)]
+    pub async fn import_from_args(args: PoolArgs) -> Result<Lvs, Error> {
+        if args.disks.len() != 1 {
+            return Err(Error::Invalid {
+                source: Errno::EINVAL,
+                msg: format!(
+                    "invalid number {} of devices {:?}",
+                    args.disks.len(),
+                    args.disks
+                ),
+            });
+        }
+
+        // default to aio if no specific driver scheme is specified
+        let disks = args
+            .disks
+            .iter()
+            .map(|d| {
+                if Url::parse(d).is_err() {
+                    format!("aio://{}", d)
+                } else {
+                    d.clone()
+                }
+            })
+            .collect::<Vec<_>>();
+
+        let parsed = uri::parse(&disks[0]).map_err(|e| Error::InvalidBdev {
+            source: e,
+            name: args.name.clone(),
+        })?;
```

Comment:
What about having a common parse function that to use aio::// if the "raw disk" fails to parse and use it everywhere we want to try aio to avoid code duplication?

Reply:
Maybe even the initial check for disk len 1 can be made common since that is also used elsewhere.

Reply:
`uri::parse(&disk)` returns a object of type `Box<dyn BdevCreateDestroy<Error = NexusBdevError>>` which I am not sure we should be included in lvs_pool
Keeping this part separate for now.